Makefile

```diff
@@ -74,7 +74,7 @@ clean:
 	-rm -rf .paasta
 
 yelpy: ## Installs the yelp-internal packages into the default tox environment
-	.tox/py37-linux/bin/pip-custom-platform install -i https://pypi.yelpcorp.com/simple -r yelp_package/extra_requirements_yelp.txt -r ./extra-linux-requirements.txt
+	.tox/py37-linux/bin/pip-custom-platform install -i https://pypi.yelpcorp.com/simple -r yelp_package/extra_requirements_yelp.txt
```

Comment:
It seems like the proper thing would be to put this in the tox file?

Reply:
(not that this is something required here, but seeing `.tox/py37-linux/bin/pip-custom-platform install` makes me cringe. It would be tricky to configure the alternative PyPI server without having a completely duplicated `tox` target)

Reply:
yea, i've been meaning to try to clean this repo's toxfile up when i have a sec :/

moving this to an optional step in our tox sounds good tho - i always get annoyed when i recreate my venv and forget to run this target :p

Reply:
wow, having yelpy deps available broken a bunch of tests ;_;